django/contrib/admin/sites.py

```diff
@@ -440,6 +440,8 @@ def _build_app_dict(self, request, label=None):
 
             info = (app_label, model._meta.model_name)
             model_dict = {
+                'model': model,
+                'model_admin': model_admin,
```

Comment:
Do you need a `model_admin`? The ticket only mentions a model instance (`model` is a class not an instance). I'm not sure how a model admin class can be used in templates :thinking:

Reply:
Yes, this is not needed